internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/ingestionHeartbeat/IngestionHeartBeatTest.java

```diff
@@ -109,15 +112,6 @@ public void setUp() {
     this.parentController = parentControllers.get(0);
   }
 
-  @AfterTest(alwaysRun = true)
-  public void cleanupStore() {
-    String parentControllerUrl = parentController.getControllerUrl();
-    try (ControllerClient parentControllerClient =
-        new ControllerClient(multiRegionMultiClusterWrapper.getClusterNames()[0], parentControllerUrl)) {
-      parentControllerClient.disableAndDeleteStore(storeName);
-    }
-  }
-
```

Comment:
Kind of. The tests in this class were failing due to the new changes. So, while fixing this, I noticed that it works weirdly.

It uses a global variable just for the purposes of cleanups after the test runs. This deleteion can also fail and cause test flakiness. So, I removed it because it wasn't really needed